site/landing/content/privacy-policy.md

```diff
@@ -1,8 +1,4 @@
----
-title: "Website Privacy and Cookies Policy"
-aliases:
-  - /privacy/
----
```

Comment:
These pages are still Hugo, is it okay to remove the frontmatter? I think the `title:` is inserted into the browser tab title

Reply:
Yep good point, I've just push a change re-adding the `title` frontmatter for all the static pages.

Reply:
Testing locally, this is indeed needed for a meaningful message in the tab title.